src/components/Pagination/Pagination.js

```diff
@@ -1,38 +1,32 @@
 import React from "react";
 import "../styles/pagination.css";
+import classNames from "classnames";
 
 function Pagination({ currentPage, handlePageChange, totalItem, limit }) {
+  const pageIndexes = [currentPage];
+
+  for (let i = 1; i <= 2; i++) {
```

Comment:
What does "2" stand for?

Reply:
It stand for next 2 page for previous 2 page from current page. Should i use constant for it ?